be/src/runtime/memory/mem_tracker_limiter.cpp

```diff
@@ -459,10 +468,18 @@ int64_t MemTrackerLimiter::free_top_overcommit_query(int64_t min_free_mem,
         max_pq.pop();
     }
     if (!usage_strings.empty()) {
-        LOG(INFO) << "Process GC Free Top Memory Overcommit " << TypeString[type] << ": "
+        LOG(INFO) << "Process GC Free Top Memory Overcommit " << type_string(type) << ": "
                   << join(usage_strings, ",");
     }
     return freed_mem;
 }
 
+void MemTrackerLimiter::add_to_task_group(taskgroup::TaskGroupPtr task_group) {
+    if (_task_group) {
```

Comment:
How about add DCHECK(_task_group == nullptr) here?
mem_tracker's task_group should be set only once.